views/js/GoogleAnalyticActionLib.js

```diff
@@ -156,7 +157,7 @@ var GoogleAnalyticEnhancedECommerce = {
 
 		//this.add(Product);
 		ga('ec:setAction', 'purchase', Order);
-		ga('send', 'pageview', {
+		ga('send', 'event','Transaction','purchase', {
```

Comment:
analytics need 'send' in order to actually send info to google server, otherwise any setAction or an yother command, wont ever reach google server.
The example in that link is just that, an example.
You can try with the send 'event' instead of the send 'pageview' and you'll see in your anlytics 'Real Time' that the transactions are sent as they should.